Statically linked pre-built binaries

Currently, only `coreutils` provide fully statically linked pre-built [binaries](https://github.com/uutils/coreutils/releases/download/0.0.26/coreutils-0.0.26-x86_64-unknown-linux-musl.tar.gz).

This is immensely useful and allows for increased adoption of this project with alternative distros.

Kindly request similar musl-based, statically-linked binaries for `findutils` and `diffutils`. Thank you!

and for diffutils:
```

   Compiling diffutils v0.4.2 (/home/runner/work/diffutils/diffutils)
error: linking with `cc` failed: exit status: 1
  |
  = note: LC_ALL="C" PATH="/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin:/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin/self-contained:/snap/bin:/home/runner/.local/bin:/opt/pipx_bin:/home/runner/.cargo/bin:/home/runner/.config/composer/vendor/bin:/usr/local/.ghcup/bin:/home/runner/.dotnet/tools:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin" VSLANG="1033" "cc" "/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crt1.o" "/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crti.o" "/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crtbegin.o" "/tmp/rustcrTDZPH/symbols.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.1hb06srr1ch4rnko.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.addr2line-686d787984217b59.addr2line.ea02d2ccd97bcad9-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.adler-193dd2a79ef160a9.adler.d1ab4299a0a3a566-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.00.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.01.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.02.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.03.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.04.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.05.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.06.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.07.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.08.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.09.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.10.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.11.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.12.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.13.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.14.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.aho_corasick-e58591dc830112a4.aho_corasick.3d94e5a655f64e69-cgu.15.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.alloc-d02027f3bdb9876a.alloc.eeea932699d00e4c-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.cfg_if-b1bae776a6e5677d.cfg_if.f0b2cf4390f56169-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.00.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.01.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.02.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.03.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.04.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.05.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.06.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.07.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.08.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.09.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.10.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.11.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.12.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.13.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.14.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.chrono-63134c1ba871c5ae.chrono.3c06b202e4d1652d-cgu.15.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.core-8135530b14978e6d.core.add8727e2c2b2dee-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diff-80f917b161bcb8eb.diff.726104c30b59fd23-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diff-80f917b161bcb8eb.diff.726104c30b59fd23-cgu.1.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diff-80f917b161bcb8eb.diff.726104c30b59fd23-cgu.2.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.00.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.01.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.02.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.03.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.04.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.05.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.06.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.07.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.08.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.09.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.10.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.11.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.12.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.13.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.14.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.diffutils.b8759b44f6a11dd0-cgu.15.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.gimli-514530b69e03e0ef.gimli.985b22b01140fbb9-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.hashbrown-e7a3685c0bbd0c42.hashbrown.df7242418d798e56-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.iana_time_zone-546c62cbbc5ddba1.iana_time_zone.5f654a4d4c93d060-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.iana_time_zone-546c62cbbc5ddba1.iana_time_zone.5f654a4d4c93d060-cgu.1.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.iana_time_zone-546c62cbbc5ddba1.iana_time_zone.5f654a4d4c93d060-cgu.2.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.iana_time_zone-546c62cbbc5ddba1.iana_time_zone.5f654a4d4c93d060-cgu.3.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.libc-d09271687f61e1fb.libc.c9b063bec0e8678c-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.memchr-02b11ddef9778e51.memchr.6179aff5c5b5f749-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.memchr-02b11ddef9778e51.memchr.6179aff5c5b5f749-cgu.1.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.memchr-64f6783c29437be0.memchr.299d4d4ad9f531e0-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.miniz_oxide-1ab977cb49e9f850.miniz_oxide.25fd3f1b34ce8b7f-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.num_traits-ee10e737077aa4fd.num_traits.2de90cc72fcd4a08-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.object-4cacc5fa8a27ecf8.object.9a082525b693912a-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.panic_unwind-43c077dbaac5c651.panic_unwind.337bb689ae04db2f-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex-2f8c5c501d04ae8b.regex.94cbea930340ac0d-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex-2f8c5c501d04ae8b.regex.94cbea930340ac0d-cgu.1.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex-2f8c5c501d04ae8b.regex.94cbea930340ac0d-cgu.2.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex-2f8c5c501d04ae8b.regex.94cbea930340ac0d-cgu.3.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex-2f8c5c501d04ae8b.regex.94cbea930340ac0d-cgu.4.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex-2f8c5c501d04ae8b.regex.94cbea930340ac0d-cgu.5.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex-2f8c5c501d04ae8b.regex.94cbea930340ac0d-cgu.6.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex-2f8c5c501d04ae8b.regex.94cbea930340ac0d-cgu.7.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex-2f8c5c501d04ae8b.regex.94cbea930340ac0d-cgu.8.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.00.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.01.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.02.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.03.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.04.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.05.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.06.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.07.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.08.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.09.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.10.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.11.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.12.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.13.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.14.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_automata-03a086896668fc86.regex_automata.aa172d22e028bada-cgu.15.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.00.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.01.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.02.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.03.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.04.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.05.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.06.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.07.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.08.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.09.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.10.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.11.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.12.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.13.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.14.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.regex_syntax-1da6fa5712d79298.regex_syntax.8cf08b0c84ab214-cgu.15.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.rustc_demangle-6d8857037a7ba1ac.rustc_demangle.89c8b96e19d3a0b6-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.rustc_std_workspace_alloc-38ecd594301af8ea.rustc_std_workspace_alloc.4f0b84efb8ac874c-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.rustc_std_workspace_core-d87b0cd085418983.rustc_std_workspace_core.8443752654e50596-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.same_file-557c59f6a91222cd.same_file.3decd2990dca2381-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.std-3aedbc8e8c0e1607.std.f516b7b62bf5fd-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.std_detect-87900d26b9dcad12.std_detect.7cbff7f4f931deb7-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.unicode_width-2771e048cd06cf20.unicode_width.ffebf0347c21af26-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c.unwind-8dea20a203d09b19.unwind.cdc344d5994e5e4-cgu.0.rcgu.o.rcgu.o" "-Wl,--as-needed" "-L" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps" "-L" "/home/runner/work/diffutils/diffutils/target/dist/deps" "-L" "/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib" "-Wl,-Bstatic" "-lunwind" "-lc" "/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib/libcompiler_builtins-08c46ed993ccb312.rlib" "-Wl,-Bdynamic" "-Wl,--eh-frame-hdr" "-Wl,-z,noexecstack" "-nostartfiles" "-L" "/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib" "-L" "/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained" "-o" "/home/runner/work/diffutils/diffutils/target/aarch64-unknown-linux-musl/dist/deps/diffutils-23e76ff45708718c" "-Wl,--gc-sections" "-static" "-no-pie" "-Wl,-z,relro,-z,now" "-Wl,-O1" "-nodefaultlibs" "/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crtend.o" "/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crtn.o"
  = note: /usr/bin/ld: /home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crt1.o: Relocations in generic ELF (EM: 183)
          /usr/bin/ld: /home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crt1.o: Relocations in generic ELF (EM: 183)
          /usr/bin/ld: /home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crt1.o: Relocations in generic ELF (EM: 183)
          /usr/bin/ld: /home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crt1.o: Relocations in generic ELF (EM: 183)
          /usr/bin/ld: /home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crt1.o: Relocations in generic ELF (EM: 183)
          /usr/bin/ld: /home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crt1.o: Relocations in generic ELF (EM: 183)
          /usr/bin/ld: /home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crt1.o: Relocations in generic ELF (EM: 183)
          /usr/bin/ld: /home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crt1.o: error adding symbols: file in wrong format
          collect2: error: ld returned 1 exit status
```

--- a/dist-workspace.toml
+++ b/dist-workspace.toml
@@ -10,7 +10,7 @@ ci = "github"
 # The installers to generate for each app
 installers = ["shell"]
 # Target platforms to build apps for (Rust target-triple syntax)
-targets = ["aarch64-apple-darwin", "x86_64-apple-darwin", "x86_64-unknown-linux-gnu", "x86_64-pc-windows-msvc"]
+targets = ["aarch64-apple-darwin", "x86_64-apple-darwin", "x86_64-unknown-linux-gnu", "x86_64-unknown-linux-musl", "aarch64-unknown-linux-musl", "x86_64-pc-windows-msvc"]
 # Which actions to run on pull requests
 pr-run-mode = "plan"
 # Path that installers should place binaries in
@@ -22,3 +22,5 @@ allow-dirty = ["ci"]
 
 [dist.github-custom-runners]
 x86_64-unknown-linux-gnu = "ubuntu-latest"
+x86_64-unknown-linux-musl = "ubuntu-latest"
+aarch64-unknown-linux-musl = "ubuntu-24.04-arm"

